cli/tests/integration/lsp_tests.rs

```diff
@@ -1444,11 +1446,11 @@ fn lsp_hover_change_mbc() {
       "range": {
         "start": {
           "line": 2,
-          "character": 13,
+          "character": 15,
         },
         "end": {
           "line": 2,
-          "character": 14,
+          "character": 16,
```

Comment:
The numbers specified for this test were previously wrong. Please double check what I've done here. Everything is 0-indexed and the language server protocol uses utf16 character indexes to make life complicated.